### DISCUSSION TITLE
Первый светодиод

### DISCUSSION THREAD

**Discussion by Sculptonics** [vvip-68/LedPanelWiFi#24] [Вопросы и ответы]
Здравствуйте.
При финальной отладке матрицы заметил, что первый светодиод при подаче питания начинает беспорядочно излучать серии световых импульсов разного цвета. Пробовал менять ESP, ничего не поменялось. Отрисовка эффектов проходит без артефактов, фликера и задержек. Впечатление, что светодиод частично испорчен. Может такое быть?
https://github.com/vvip-68/LedPanelWiFi/assets/64962556/d1a57d9b-a3ad-46ca-80eb-1427859370b4

**Comment by vvip-68** [comment]
Нет, думаю что не может быть частично испорчен.
В версии 1.13 - это был признак не той версии ядра. В версии 1.14 - используется последнее ядро, которое уже работает правильно.
Единственно возможный вариант - превышение уровня напряжения питания выше нормы, но обычно при этом плохо себя ведут все диоды, на всей матрице. Так что я даже теряюсь в догадках о возможной причине. Здесь речь о ESP8266

**Comment by Sculptonics** [comment]
Перепроверил версию библиотеки FastLED. Стояла 3.4.0 (в версии прошивки 1.13 требование было не строгое, поэтому тогда не стал обновлять).
Установил 3.5.0. Всё работает правильно.
Спасибо!

**Comment by RRiVan** [comment]
Здравствуйте! Была проблема подобная на самодельной матрице ( 2 матрицы, симптомы одни и те же). Светился и моргал сам по себе первый  диод, и передавал дальше с ошибками, например при заливке матрицы каким либо цветом, заливал другим. снег красный и т.п.. Менял контроллер (Брал с рабочей гирлянды) , менял первый диод на другой, менял библиотеки, менял прошивки, менял БП, Впаивал понижающий  диод по питанию, ставил резистор развязки. Ничего не помогало. И решил попробовать включить готовую гирлянду вместо самодельной. Она заработала.... Взял Светодиод из нее и впаял первым в свою матрицу. Всё заработало!.
